PivotGrid: update a demo related to headerFilter (#2568)
DevExpress/devextreme-demos: JSDemos/Demos/PivotGrid/HeaderFilter/description.md

To configure a header filter, use the global [headerFilter](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/) object or a field's [headerFilter](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/headerFilter/) object. This demo specifies the following properties in the global **headerFilter** object:

- [search](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/headerFilter/search/)     

> Allows users to configure a search through field values. 

Reviewer: ```suggestion
Allows you to configure a search panel within a header filter. 
```

> 

Reviewer 2: On line 14 below:

In the last sentence, can you change "checkbox" to "check box"?

Author: Yes, I can.
